src/pages/Token/Modals/SellModal.tsx

```diff
@@ -90,7 +90,7 @@ export const AskSellModal: FC<TAskSellModalProps> = ({ onSellAuction, onSellFixP
   );
 
   const onConfirmAuctionClick = useCallback(() => {
-    if (!selectedAccount || !minStepInputValueAuction || !durationSelectValue) return;
+    if (!selectedAccount || !minStepInputValueAuction || !durationSelectValue || !inputStartingPriceValue || !Number(minStepInputValueAuction) || !Number(inputStartingPriceValue)) return;
```

Comment:
put it into variable, smth like "validation". Or make a function that will return this. You are using it in two places.